source/hardware/controllers/reloop_mixage.rst

```diff
@@ -0,0 +1,115 @@
+26      Linefader                                  Controls the respective channel's volume.                                                                                                                                    Same as non-shift function.
+33      Crossfader                                 Crossfader between channel 1 and channel 2.                                                                                                                                  Same as non-shift function.
+======  =========================================  ===========================================================================================================================================================================  ==============================================================================================================================================================================
+
+The effect buttons and knobs on the left side apply to effect rack 1, the ones on the right to effect rack 2. The numbers from the overview correspond to the product manual and mapping XML file Txx numbers.
+
+Jog wheel
+---------
+
+- With :hwlabel:`🔍` button active: Scroll through the track. While touching the top of the wheel scrolling is three times as fast.
+- With :hwlabel:`💿` button active: Touch the side of the jog wheel to nudge the deck. Touch the top of the jog wheel to scratch (either deck playing or not).
+- With the Beatloop adjust mode active: Adjusts the selected loop point.
+
+Beatloop adjust mode
+--------------------
+
+When a beatloop has been set, press :hwlabel:`SHIFT` + :hwlabel:`-LENGTH+` / :hwlabel:`BEATMOVE` to enable beatloop adjust mode. The :hwlabel:`LOOP` / :hwlabel:`IN` and :hwlabel:`RELOOP` / :hwlabel:`OUT` buttons will blink. Now you can:
+
+- Move the entire loop using the jog wheel, adjustment is three times as fast while the top of the jog wheel is touched.
+- Press :hwlabel:`LOOP` / :hwlabel:`IN` to shift the loop in point using the jog wheel. The :hwlabel:`LOOP` / :hwlabel:`IN` will blink.
+- Press :hwlabel:`RELOOP` / :hwlabel:`OUT` to shift the loop out point using the jog wheel. The :hwlabel:`RELOOP` / :hwlabel:`OUT` will blink.
+
+Hold :hwlabel:`SHIFT` and press :hwlabel:`-LENGTH+` / :hwlabel:`BEATMOVE`, or press :hwlabel:`🔍` or :hwlabel:`💿` button to leave beatloop adjust mode.
+
+User-adjustable script settings
+-------------------------------
+
+The `Reloop-Mixage.scripts.js` controller script provides the following settings:
+
+-  Scratch behaviour can be changed to a more regular, turntable-like mode by changing the setting `scratchByWheelTouch` to `true`. Note that the jog wheels are not very sensitive to touch though (sensitivity can be adjusted on the back of the controller in the MK1 version).
```

Comment:
Let's use the comment from the script:

```suggestion
-  `scratchByWheelTouch` (default `false`): change to `true` to get a more regular, turntable-like mode, i.e. scratch by touching the jog wheel instead of having to toggle the disc button. Note that the jog wheels are not very sensitive to touch though (sensitivity can be adjusted on the back of the controller in the MK1 version).
```